var/spack/repos/builtin/packages/argobots/package.py

```diff
@@ -20,6 +20,8 @@ class Argobots(AutotoolsPackage):
     git      = "https://github.com/pmodels/argobots.git"
     maintainers = ['shintaro-iwasaki']
 
+    tags = ['ecp', 'e4s']
```

Comment:
🤔 Is this actually an ECP project?

Reply:
Nor is qt, but they seem to be explicit packages in e4s... I agree the ecp tag should be dropped if the code development isn't at least partly funded by ecp.

Reply:
It appears in slides BUT not as a product.

Reply:
okay `ecp` tag removed from argobots